### DISCUSSION TITLE
How to change the audio icon

### DISCUSSION THREAD

**Discussion by mvukovic95** [black7375/Firefox-UI-Fix#666] [Q&A]
Just installed this theme and I'm liking it but I'd like to change the audio icons back to the default if possible. Is that possible?

![1](https://user-images.githubusercontent.com/32885702/224580294-1e60181f-2558-4075-a57a-9757603b47b3.png)
![2](https://user-images.githubusercontent.com/32885702/224580295-39f0c797-9aff-4830-b25a-5e3b3745318d.png)
Firefox with this theme

![3](https://user-images.githubusercontent.com/32885702/224580652-b09bfb47-3d14-44de-93ca-9baec4e5d5de.png)
![4](https://user-images.githubusercontent.com/32885702/224580654-b9a058a6-58b2-44f0-ba83-0a0d5dcc5573.png)
Firefox by default

**Comment by black7375** [comment]
Edit [your `user.js` file](https://github.com/black7375/Firefox-UI-Fix/wiki/Tutorial#edit-userjs-file).
Search and change the following values.
- `userChrome.tab.sound_hide_label` to `false`
- `userChrome.tab.sound_with_favicons` to `false`

Refer:
- [Options - Tab default](https://github.com/black7375/Firefox-UI-Fix/wiki/Options#defaults-3)

**Reply by Aleksandra0056** [reply]
Interesting, I applied these settings, then restarted Firefox, and it reverted back to `true`. Can you tell me why?

**Reply by black7375** [reply]
See tutorial guide.
https://github.com/black7375/Firefox-UI-Fix/wiki/Tutorial#edit-userjs-file

The value in `user.js` file is read again and overwritten at every restart.

**Reply by Aleksandra0056** [reply]
I applied the mentioned values, restarted, cleared startup cache, and it reverted back to `true`.

**Reply by black7375** [reply]
> Edit your user.js file.

**Reply by Aleksandra0056** [reply]
Thank you for response. Fine, I will edit.
